lib/actions/apiV2.js

```diff
@@ -436,11 +438,15 @@ export const fetchNearby = (position, radius) => {
                 lon
                 code
                 gtfsId
+                stopRoutes: routes (serviceDates: $currentServiceWeek) {
```

Comment:
I'm getting a "Unknown field argument" graphql error back when I try to view a stop

Reply:
This only works on the latest version of OTP. Are you testing against an older OTP version? I'm happy to delay merging this until newer OTP is more widespread

Reply:
Oh gotcha, yes the instance of OTP I was using to test looks like the only one of ours this would break! We can discuss that internally